ext/TrixiBaseMPIExt.jl

```diff
@@ -0,0 +1,20 @@
+# Package extension for adding MPI-based features to TrixiBase.jl
+module TrixiBaseMPIExt
```

Comment:
I distinctly remember a conversation with one of the Pkg.jl gurus (Frederik?) where it was pointed out that the name of the package extension is never shown without the package itself, thus it is best practice to just name it `PackageExt`:
```suggestion
module MPIExt
```

Reply:
As you see in my first post above, you can get messages like

```
[ Info: Precompiling TrixiBaseMPIExt [722ac391-ce67-58c0-be25-224aa07e4d5d]
```

Thus, I would like to keep it as it is.

Reply:
This naming convention is also chosen in the official docs at https://pkgdocs.julialang.org/v1/creating-packages/#Conditional-loading-of-code-in-packages-(Extensions)

Reply:
> This naming convention is also chosen in the official docs

I know. I _did_ have a conversation with them about it, but it seems you're right. Thus, I'm ok with leaving it as it is 👍